Wrapping/Generators/SwigInterface/CMakeLists.txt

```
@@ -363,28 +373,46 @@ macro(itk_end_wrap_module_swig_interface)
         )
         set(snake_case_config_file
             "${ITK_WRAP_PYTHON_SNAKE_CONFIG_DIR}/${WRAPPER_LIBRARY_NAME}_snake_case.py")
+        add_custom_command(
+        OUTPUT ${i_files} ${typedef_files} ${idx_files} ${snake_case_config_file} ${ITK_STUB_PYI_FILES}
+        COMMAND ${Python3_EXECUTABLE} ${IGENERATOR}
+          ${mdx_opts}
+          ${swig_libs}
+          -w1 -w3 -w51 -w52 -w53 -w54
+          -A protected -A private
+          -p ${PYGCCXML_DIR}
+          -g ${CASTXML_EXECUTABLE}
+          --snake-case-file "${snake_case_config_file}"
+          --interface-output-dir "${WRAPPER_MASTER_INDEX_OUTPUT_DIR}"
+          --library-output-dir "${WRAPPER_LIBRARY_OUTPUT_DIR}"
+          --submodule-order "${WRAPPER_SUBMODULE_ORDER}"
+          --pyi_dir "${ITK_STUB_DIR}"
+          DEPENDS ${ITK_WRAP_DOC_DOCSTRING_FILES} ${xml_files} ${IGENERATOR} ${typedef_in_files}
+          WORKING_DIRECTORY / # Arguments to WORKING_DIRECTORY may use generator expressions
```

Can we use something other than root (`/`) here? This might be what creates `M:\RelWithDebInfo` etc folders in my drive roots.
```suggestion
          WORKING_DIRECTORY "${ITK_DIR}/Wrapping/$<CONFIG>" # Arguments to WORKING_DIRECTORY may use generator expressions
```